orthanc/orthanc-raw/plugin/pixl.py

```diff
@@ -169,27 +136,19 @@ def log_and_return_http(
 def SendResourceToAnon(output, uri, **request):  # noqa: ARG001
     """Send an existing study to the anon modality"""
     orthanc.LogWarning(f"Received request to send study to anon modality: {request}")
-    if not should_auto_route():
-        log_and_return_http(
-            output,
-            200,
-            "Auto-routing is not enabled",
-            f"Auto-routing is not enabled, dropping request {request}",
-        )
-        return
     try:
         body = json.loads(request["body"])
-        resource_id = body["ResourceId"]
+        body["Resources"]
```

Comment:
according to the [api docs for c-store](https://orthanc.uclouvain.be/api/#tag/Networking/paths/~1modalities~1{id}~1store/post) we need to specify a list of resources now that we're sending json